scripts/db_migrator.py

```diff
@@ -1170,8 +1168,6 @@ def version_4_0_2(self):
         Version 4_0_2.
         """
         log.log_info('Handling version_4_0_2')
-        if self.stateDB.keys(self.stateDB.STATE_DB, "FAST_REBOOT|system"):
```

Comment:
Changing the functionality of old and already deployed versions is not generally recommended - the db version handling should be same on all feature branches. Otherwise it creates conflicting views and also causes confusion.

I think you can leave `version_4_0_2` and `migrate_config_db_flex_counter_delay_status` untouched, and handle your new intention in a new function (as you are already doing in common ops).